assets/dfn-panel.css

```diff
@@ -0,0 +1,59 @@
+/* dfn popup panel that list all local references to a dfn */
+dfn {
+  cursor: pointer;
+}
+.dfn-panel {
+  position: absolute;
+  left: var(--left); /* set via JS */
+  top: var(--top); /* set via JS */
+  z-index: 35;
+  height: auto;
+  width: max-content;
+  max-width: 300px;
+  max-height: 500px;
+  overflow: auto;
+  padding: 0.5em 0.75em;
+  font: small Helvetica Neue, sans-serif, Droid Sans Fallback;
+  background: #dddddd;
+  color: black;
+  border: outset 0.2em;
+}
+.dfn-panel * {
+  margin: 0;
+  padding: 0;
+  text-indent: 0;
+}
+.dfn-panel > b {
+  display: block;
+}
+.dfn-panel a[href] {
+  color: black;
+}
+.dfn-panel a[href].self-link {
+  display: inline-block;
```

Comment:
With `display: inline-block`, longer self-link text is slightly harder to select (block padding)
![image](https://user-images.githubusercontent.com/8426945/70704631-6890ab00-1cf8-11ea-8fed-1cc8f17573cd.png)

Without inline-block, small text wrap around unnecessarily
![image](https://user-images.githubusercontent.com/8426945/70704415-f1f3ad80-1cf7-11ea-9560-cab5c7d9ca4c.png)

`white-space: nowrap` helps with small text:
![image](https://user-images.githubusercontent.com/8426945/70704455-0768d780-1cf8-11ea-9099-1f4fd5295ade.png)

but adds scroll on longer texts:
![image](https://user-images.githubusercontent.com/8426945/70704529-28c9c380-1cf8-11ea-80cc-b559b8e1bb93.png)

Anyone with better ideas?

Reply:
Scrollable might be ok. People generally click, and then copy the URL. We should think of how to avoid showing the fragment entirely in the future. We could just have a “#”.

Reply:
TC39 calls this label as just "Permalink". Maybe we can use that

Reply:
I'm happy with that.